engine/src/CalciteInterpreter.cpp

```diff
@@ -106,10 +106,16 @@ std::vector<std::unique_ptr<ral::frame::BlazingTable>> execute_plan(std::vector<
 			// useful when the Algebra Relacional only contains: ScanTable (or BindableScan) and Limit
 			query_graph->check_for_simple_scan_with_limit_query();
 
+			size_t max_kernel_run_threads = 4; //default
```

Comment:
default should be higher. should not be sset here and in another place like i see at the bottom.